Jellyfin.Plugin.PushOverNotifications/Api/ServerApiEntryPoints.cs

```diff
@@ -8,7 +8,7 @@
 using System.Threading.Tasks;
 using System.Threading;
 
-namespace Jellyfin.Plugin.PushOverNotifications.Api
+namespace Jellyfin.Plugins.PushOverNotifications.Api
```

Comment:
Root namespace should be `Jellyfin.Plugin.PushOverNotifications` (no s) for all classes